app/helpers/application_helper.rb

```diff
@@ -43,13 +43,13 @@ def publication_task_state(task)
 
   def bootstrap_class_for(flash_type)
     case flash_type
-    when :success
+    when "success"
       "alert-success" # Green
-    when :error
+    when "error"
       "alert-danger" # Red
-    when :alert
+    when "alert"
       "alert-warning" # Yellow
-    when :notice
+    when "notice"
```

Comment:
Nice catch. 

Are we at risk though that any uses of symbol may fail, e.g here: https://github.com/alphagov/manuals-publisher/blob/e01ac22466d1863486a01c81b1bf0f443bf9ad4f/app/controllers/manuals_controller.rb#L25

Reply:
Doesn't seem to be—the `class` of the `type` variable is `String`, even in the instance you mention.

I don't really understand why, but that's what my empirical testing shows…

Reply:
Oh that's a bit strange. Does that mean that this code didn't work before? since it couldn't have worked for string and symbol as far as I'm aware 🤔 

I'll just have a quick look